test/functionalTest/cases/2750_common_metrics/2750_incomingTransactions.test

```diff
@@ -30,21 +30,21 @@ brokerStart CB
 --SHELL--
 
 #
-# 01. Send five GET /version
+# 01. Send five GET /v2/entities
 # 02. Ask for Metrics, see 5 Incoming Transactions
 #
 
-echo "01. Send five GET /version"
-echo "=========================="
-orionCurl --url /version --tenant S1 --servicePath /SP1
+echo "01. Send five GET /v2/entities"
+echo "=============================="
+orionCurl --url /v2/entities --tenant S1 --servicePath /SP/entities1
```

Comment:
For the future, better to use `/SP_entities1`, as hierarchical SPs are not an use case at IoTP.

For this PR, it doesn't worth the time to change it. NTC.